app/Providers/ScheduleServiceProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Illuminate\Console\Scheduling\Schedule;
+
+class ScheduleServiceProvider extends ServiceProvider
```

Comment:
Is this auto-registered somehow?

Reply:
No, it's registerd in app.php around line 173.

Reply:
Ah right, yes, missed that somehow :+1: